### PR TITLE
chapter_12

### PR DESCRIPTION
**Pull request by muizzuddinyafiq**
ready

### CODE REVIEW

**Comment by muizzuddinyafiq** [comment]
ready

File: app/assets/stylesheets/password_reset.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the PasswordReset controller here.
```
Comment (ghost) [inline app/assets/stylesheets/password_reset.scss]: delete this file please

File: app/helpers/password_reset_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module PasswordResetHelper
```
Comment (ghost) [inline app/helpers/password_reset_helper.rb]: delete this file please

File: test/controllers/password_reset_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+require "test_helper"
```
Comment (ghost) [inline test/controllers/password_reset_controller_test.rb]: you dont need to do test. Please delete this file

**Review by ghost — COMMENTED** [review]
cmt

**Comment by muizzuddinyafiq** [comment]
ready

**Comment by ngocvt-0484** [comment]
merged

**Pull request merged**